lib/sycamore/sycamore/data/element.py

```diff
@@ -159,8 +159,7 @@ def __init__(
         self.data["properties"]["rows"] = rows
         self.data["table"] = table
         self.data["tokens"] = tokens
-        if self.data["table"]:
-            self.data["text_representation"] = self.data["table"].to_csv()
+        self.data["text_representation"] = None
```

Comment:
Leaving this unset is fine, too, and matches the previous code if table is not provided.  If we use self.data.get("text_representation") everywhere, there's no difference.  Personally, I like to err on the side of less code, but there's also an argument for "full initialization".

Reply:
Have removed it to match previous behavior